skia-safe/src/gpu/context.rs

```diff
@@ -142,7 +142,15 @@ impl RCHandle<GrContext> {
         self
     }
 
-    // TODO: performDeferredCleanup()
+    pub fn perform_deferred_cleanup(&mut self, not_used_millis: i32) -> &mut Self {
```

Comment:
So far I've preferred `std::time::Duration` to represent untyped time duration values on the Rust side. What do you think, can we ignore the performance a bit to get more type safety here?

Reply:
Totally. I was going to do that, but then thought just reflect 1:1 the Skia's APIs.